pages/metadata.md

```diff
@@ -0,0 +1,169 @@
+backronym meaning “YAML Ain’t Markup
+Language.”](https://en.wikipedia.org/wiki/YAML)
+
+Here are the required fields:
+
+-   Title
+-   Authors
+-   Tags
+-   Excerpt
+-   Description
+
+There are also a few optional fields:
+
+-   Date
+-   Image
+
+Here is what each required field means and some guidelines for filling
+them out:
+
+Title
+-----
+
+The plain-text title of your post, surrounded by quotation marks. This
+will be displayed prominently above the post, will show up in browser
+tabs, and will be included in "share text" when the link appears on
+Twitter, Facebook, and other social media platforms.
+
+Titles should follow [sentence
+case](https://pages.18f.gov/content-guide/capitalization/) (only the
+first word should be capitalized), and they should tell the reader the
+most important piece of information in the story. If you're launching a
+product, communicate that in the title instead of something less
+specific like *Continuing 18F’s commitment to open data.* If the title
+of your product is descriptive of its function (for example, College
+Scorecard) then it’s good to include it in the title. If it’s less clear
+(for example, Midas) then it’s better to explain what the product does
+and save the name for the first paragraph.
+
+Authors
+-------
+
+A list of handles of teammates involved in authoring the post. They must
+all be lowercase, and must match a name that appears in
+[_data/authors.yml](https://github.com/18F/18f.gsa.gov/blob/staging/_data/authors.yml).
+If your author doesn’t exist, [add it to the
+file](https://github.com/18F/18f.gsa.gov/edit/staging/_data/authors.yml).
+Names listed here will automatically be added to the post’s byline. In
+addition, if the author has a bio on 18f.gsa.gov, posts tagged with
+their name will automatically link to their bio. See our [authorship
+section]({{
+site.baseurl }}/content-guidelines/) for more information on how to choose authors.
+
+Tags
+----
+
+A list of tags to associate with the post. These will appear, linked,
+next to the post and will take readers to other posts that have this
+tag. Sentences (for example, "how we work") are fine — there's no need to jam
+phrases into one word. Follow our [tagging
+guide]({{
+site.baseurl }}/tagging/)
+when choosing tags.
+
+Excerpt
+-------
+
+A short plain-text snippet of the post to use as the "[nut
+graf](https://en.wikipedia.org/wiki/Nut_graph)" that appears on the
+home page. This should be the most important part of your post. Think of
+it as what you would tell a friend if they asked you about your post. A
+good excerpt is usually no more than two sentences long. No Markdown
+allowed.
+
+Description
+-----------
+
+A short, plain-text description of your post, surrounded with quotation
+marks. No Markdown or HTML allowed. This can be the same as the excerpt
+or it can be a sentence or two that you feel represents your post well.
+It may appear next to your post on social media and other places that
+fetch article metadata.
```

Comment:
Can't remember if we settled this in the Google Doc before it all got ported HTML is technically allowed (like Jekyll will render it) in the excerpt and description fields. Markdown isn't.